controllers/drplacementcontrol.go

```diff
@@ -103,6 +103,8 @@ func (d *DRPCInstance) RunInitialDeployment() (bool, error) {
 
 	const done = true
 
+	d.reportEvent(d.getLastDRState())
```

Comment:
Why is this here? `setDRState` will always call it once the state changes.

Reply:
Yes. Correct. I noticed that there are many unknown events reported by default for example
2022-11-23T16:16:16.083+0530    DEBUG   events  next state not known    {"type": "Warning", "object": {"kind":"DRPlacementControl","namespace":"app-namespace","name":"app-volume-replication-test","uid":"a79c9853-b867-46e2-a11f-e7b7b3ab2859","apiVersion":"ramendr.openshift.io/v1alpha1","resourceVersion":"542"}, "reason": "unknown state"}
I tried to add more info with event DRPCInitiating instead of unknown state which kind of confusing.  That's the reason I called the reportEvent. we still have some intermediate state where 'unknown state' reported. 
If this is making more verbose I can remove and let it be as it was with couple of unknown state events.
2022-11-23T16:16:16.083+0530    DEBUG   events  Preparing for exectuion {"type": "Normal", "object": {"kind":"DRPlacementControl","namespace":"app-namespace","name":"app-volume-replication-test","uid":"a79c9853-b867-46e2-a11f-e7b7b3ab2859","apiVersion":"ramendr.openshift.io/v1alpha1","resourceVersion":"542"}, "reason": "DRPCInitiating"}